mlflow/utils/requirements_utils.py

```diff
@@ -334,7 +334,7 @@ def _capture_imported_modules(model_uri, flavor):
 
 def _init_modules_to_packages_map():
     global _MODULES_TO_PACKAGES
-    if _MODULES_TO_PACKAGES is None and _PACKAGES_TO_MODULES is None:
+    if _MODULES_TO_PACKAGES is None:
```

Comment:
`_init_modules_to_packages_map` doesn't need to worry about `_PACKAGES_TO_MODULES`.